bin/run_brick.rb

```diff
@@ -78,7 +78,7 @@ def handle_error(params, log, brick_type, error, error_message)
 
   log.info "action=#{brick_type}_execution status=start commit_hash=#{commit_hash} execution_id=#{execution_id}"
   brick_pipeline.call(params)
-  update_execution_result(execution_result_log_params, log, brick_type, ExecutionStatus::OK, '')
+  update_execution_result(execution_result_log_params, log, brick_type, ExecutionStatus::OK, '') unless params['EXECUTION_RESULT_COMPLETED']
```

Comment:
we should remove method update_execution_result in run_brick.rb, switch to use ExecutionResultMiddleware